assessments/0011-keda/keda-analysis.md

```diff
@@ -0,0 +1,416 @@
+- Azure Functions and Queues
+- Azure Functions and Kafka on Openshift 4
+- Azure Storage Queue with ScaledJob
+
+These pages list installing KEDA as a prerequisite. Taken together, "Deploying KEDA" and the scenarios in the repo make a complete Getting Started workflow, but they are in two separate places and the scenarios are not findable from the website.
+
+There are also numerous examples available in the [samples repo](https://github.com/kedacore/samples).
+
+KEDA does not require documentation for multiple **operating systems**.
+
+The [Operate](https://keda.sh/docs/2.13/operate/) topic in the TOC provides instructions for using KEDA and the scenarios in the repo provide user instructions. However, new users starting on the website might not **know where to go after installation**. A more explicit **getting started workflow** would be helpful.
+
+There is no obvious **new user signpost** in the documentation. The closest is *Deploying KEDA*, which again just gives installation instructions but no roadmap.
+
+There is easily **copy-pastable** content for CLI input where appropriate, including in the installation instructions.
+
+### Content maintainability & site mechanics
+
+The documentation is **Searchable** using functionality provided by the site generation engine.
+
+There do not seem to be any plans for **localization**. Docsy has **support for multiple languages** in case that changes.
+
+The documentation is **versioned**. The repo contains a separate folder for each version.
+
+### Content creation processes
+
+The procedure for **building and releasing a new version** is documented in the doc repo README file.
+
+The code contribution instructions contain brief **documentation creation and update** instructions.
+
+The MAINTAINERS doc in the website repo points to the MAINTAINERS doc in the main repo, which does not tell who is **responsible for documentation pull requests**. The website does not have a **clear owner or maintainer**.
+
+### Inclusive language
+
+The documentation includes some examples of [**non-inclusive language**](https://inclusivenaming.org/). For example:
+- "easily", "simple", "simply", etc.
+  - https://keda.sh/docs/2.13/deploy/
+  - https://keda.sh/docs/2.13/deploy/#uninstall-3
+  - https://keda.sh/docs/2.13/concepts/scaling-deployments/#triggers
+- "master" node
+  - https://keda.sh/docs/2.13/troubleshooting/#google-kubernetes-engine-gke
+  - https://keda.sh/docs/2.13/scalers/redis-sentinel-lists/#authentication-parameters
+
+## Recommendations
+
+### Information architecture
+
+Reorganize the Table of Contents. Rename "The KEDA Documentation"; the name is misleading since it implies that it contains the entire documentation set. I'd suggest the following changes:
+- Rename "The KEDA Documentation" to "Getting Started".
+- Create a "Reference" topic at the end of the ToC.
+- Move the FAQ to the Reference section.
+- Add a glossary to the Reference section.
+- Rename "Operate" to "User Guide".
```

Comment:
Mind elaborating on this? How do we ensure that operators know to look here?

Reply:
I think I said somewhere that, since there's essentially only one user role, "User Guide" was appropriate, but you're right, I don't think there's a reason not to rename it "Operator's Guide". 

Also, I'm a big proponent of cross-linking, so I'd recommend providing links to the Operator's Guide from everywhere it might be relevant -- including but not limited to the end of the "Getting Started" workflow and the Contributor's guidelines in the repo.